changelogs/unreleased/2320-tareqhs

```diff
@@ -0,0 +1 @@
+allow ignoring label in Schedule when copying to Backup via `--ignore-schedule-label`
```

Comment:
For clarity, let's pluralize "labels":

"allow ignoring labels in Schedule when copying to Backup via `--ignore-schedule-labels`"